filip/models/ngsi_v2/context.py

```diff
@@ -8,7 +8,7 @@
 from aenum import Enum
 from pydantic import field_validator, ConfigDict, BaseModel, Field, model_validator
 from pydantic_core.core_schema import ValidationInfo
-
+from pydantic.types import OnErrorOmit
```

Comment:
I see that OnErrorOmit is introduced from Pydantic version 2.6.0 onwards, could you please also adapt version changes in setup.py file?